Camera/ListViewViewController.swift

```diff
@@ -116,13 +108,15 @@ class ListViewViewController: UIViewController, UITableViewDataSource, UITableVi
         alertController.addAction(cancelAction)
         
         let destroyAction = UIAlertAction(title: "Yep, delete them.", style: .Destructive) { (action) in
+            let realm = try! Realm()
+            try! realm.write {
+                realm.deleteAll()
```

Comment:
This doesn't work properly. Doing realm.deleteAll() is deleting the file references in the DB but we still need to delete the files or the app bundle will get huge.